robot-name/hemartin/robot_name.go

```diff
@@ -0,0 +1,54 @@
+package robotname
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+const letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+const numberUpperLimit = 999
+
+var robotNumber = 0
+var robotNames = func() []string {
+	names := []string{}
+	for _, firstLetter := range letters {
+		for _, secondLetter := range letters {
+			for number := 0; number <= numberUpperLimit; number++ {
+				name := fmt.Sprintf("%c%c%03d", firstLetter, secondLetter, number)
+				names = append(names, name)
+			}
+		}
+
+	}
+	return names
+}()
+
+func init() {
```

Comment:
Using init() is the evil! :-)

Reply:
Hahaha yes, I wish it called a `Setup()` method.